src/databricks/labs/ucx/source_code/queries.py

```diff
@@ -41,7 +41,7 @@ def _catalog(table):
             return table.catalog
         return 'hive_metastore'
 
-    def apply(self, code: str) -> str:
+    def apply(self, code: str, *, use_schema: str = "") -> str:
```

Comment:
You are not allowed to change the Fixer interface.

Why do you need a parameter here? Please explain. See also:

- https://github.com/databrickslabs/ucx/issues/1215